src/os/bluestore/BlueStore.cc

```diff
@@ -4395,6 +4411,31 @@ bool BlueStore::is_rotational()
   return rotational;
 }
 
+bool BlueStore::is_journal_rotational()
+{
```

Comment:
assert(bluefs);
return bluefs->wal_is_rotational();

and drop the rest of this method... pretty sure we can ignore the case where this gets called before mount()?